commit 533f494f46d65be366aee3e9973b674320e46bc8
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1336698215 -0500

    Fix knotes not appearing on the desktop when a session is restored.
    Thanks to Slavek Banko.
    This resolves bug report 987.

diff --git a/knotes/knote.cpp b/knotes/knote.cpp
index 182aee8..08c24b8 100644
--- a/knotes/knote.cpp
+++ b/knotes/knote.cpp
@@ -765,7 +765,7 @@ void KNote::slotClose()
     m_editor->clearFocus();
     m_config->setHideNote( true );
     m_config->setPosition( pos() );
-    m_config->writeConfig();
+
     // just hide the note so it's still available from the dock window
     hide();
 }
